march_hardware_builder/src/HardwareBuilder.cpp

```diff
@@ -43,8 +43,9 @@ march4cpp::MarchRobot HardwareBuilder::createMarchRobot(YAML::Node marchRobotCon
   ROS_INFO_STREAM("marchRobotConfig " << marchRobotConfig << "\n");
   if (marchRobotConfig["powerDistributionBoard"].Type() != YAML::NodeType::Undefined)
   {
-    march4cpp::PowerDistributionBoard powerDistributionBoard =
-        createPowerDistributionBoard(marchRobotConfig["powerDistributionBoard"]);
+    march4cpp::PowerDistributionBoard powerDistributionBoard = createPowerDistributionBoard(marchRobotConfig["powerDist"
```

Comment:
I think this is not the style you want?

Reply:
Do you not think it is beautiful :p

Fixed!